mapbook-iOS.xcodeproj/project.pbxproj

```diff
@@ -258,6 +258,7 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
+				English,
```

Comment:
Having `developmentRegion` as `English ` generates a Buildtime issue. Rather than adding the deprecated `English` identifier to `knownRegions`, I would suggest changing `developmentRegion` to `en`, which is already in `knownRegions`.